### ISSUE TITLE
[Tracking] Accessibility Testing and Remediation

### ISSUE THREAD

**Issue by LauRoxx** [Availity/element#212]
Tracking violations reported by Availity using Level Access testing tools (Level Access Extension, Access Assistant, Continuum Explorer Pro) and Storybook (Axe). Does not account for other testing tools like Wave.

### Level Access Reported Violations
- [x] #213
- [x] #392
- [ ] This title contains text with a luminosity contrast ratio of 2.639, which is below 4.5:1
- [ ] This DIV is focusable but lacks a role attribute

### Axe Reported Violations
- [ ] [Search/SearchBy] Form Elements Should Have Visible Label
